pkg/apis/submariner.io/v1/types.go

```diff
@@ -78,8 +78,13 @@ type EndpointSpec struct {
 	NATEnabled    bool              `json:"nat_enabled"`
 	Backend       string            `json:"backend"`
 	BackendConfig map[string]string `json:"backend_config,omitempty"`
+	Ports         map[string]string `json:"ports,omitempty"`
```

Comment:
@sridhargaddam tom was suggesting that we use NATDiscoveryPort as a field here *int instead.

And leave the backend ports in "BackendConfig"

does that sound ok for you?

Reply:
If I understood it correctly, any custom IPsec/tunnel ports will be configured in BackendConfig. If so, yes, its fine for me.

Reply:
:+1 thanks for the ack